src/zenml/steps/utils.py

```diff
@@ -150,6 +151,8 @@ def create_component_class(step: "BaseStep") -> Type[_SimpleComponent]:
     Returns:
         The component class.
     """
+    from tfx.dsl.component.experimental.decorators import _SimpleComponent
```

Comment:
For future curiosity: what is this and what does importing it fix? Looks like magic!

Reply:
This import drags along the whole tfx.experimental module, which loads a lot of other modules. Among them, cryptography, which, if loaded in memory, won't allow any package updates through `zenml integration install` on Windows.

The root problem is that running _any_ CLI command (`zenml integration *` included) imports all the modules from _all_ CLI commands in memory, which 1. takes time and 2. makes uninstalling/upgrading some packages with pip impossible on Windows.